module-2/assignments/activity1.xml

```diff
@@ -0,0 +1,30 @@
+<cards>
+    <card>
+        <name>Rorie Demanche</name>
+        <phone>+1 (710) 153-4949 (home)</phone>
+        <phone>+1 (503) 319-3558 (work)</phone>
+        <phone>+1 (182) 748-1667 (mobile)</phone>
```

Comment:
element itself should be telling what type of phone is stored in it, not the content inside the element